extension/framework/goplugin/runner/runner.go

```diff
@@ -224,7 +225,9 @@ func (testRunner *TestRunner) runSingle(t ginkgo.GinkgoTestingT, reporter *Repor
 	}
 
 	// prepare test
-	test(env)
+	mockEnv.Reset()
```

Comment:
I think this is unceccesary. One reset after each test is enough. At the beginning, env should be clean.

Reply:
Currently this is required - without it every first test in the suite will fail. Added todo comment to remove it in the future.